djs-bot/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"description": "",
 	"main": "index.js",
-	"version": "5.4.3",
+	"version": "5.4.4",
```

Comment:
another bump is another release, you should keep it at .3 and we will only have one release later since all your PR are only refactors. We don't need 2 releases for some patches in one day

Reply:
wait latest release is 5.4.1 wtf. The new version should stay at 5.4.2 until it's actually released smh

Reply:
We have already pushed a couple of PRs.. What version should I actually set in `package.json`?

Reply:
well ima let @BioCla make the releases for these version bumps since he's the one merged all your PR

Reply:
We'll release directly with 5.4.3 after everything is merged. The bumps are correct to be 2 since we can consider one for the new command and the other for the refactoring

Reply:
New command is new feature and if we actually follow semver the minor version should be incremented, not the patch version.

Well as long as it works, not a problem for me either way